### DISCUSSION TITLE
'Like' Public Patches

### DISCUSSION THREAD

**Discussion by jaalibandar** [cables-gl/cables_docs#218] [Feature Requests] [answered]
The current way of 'like'-ing(hitting the like button) stuff on the website is that you need to open the patch page - it would be nice if maybe you can 'like' patches from other places as well - for example 'liking' it on the patch thumbnail itself 

The current navigation makes it really hard to 'like' stuff as you can only do it at the public patch page for every patch - so it becomes a task instead of being just something you feel about something and capturing it :P

**Comment by steam0r** [comment, marked as answer]
Hey, we added a small infobox in the parampanel of the cables editor on dev.cables.gl, that also includes like and follow functions.
you can see it in patches that are owned by someone else. check it out!

**Reply by jaalibandar** [reply]
Not all of them have it currently (snapshot from the dev version)
![image](https://user-images.githubusercontent.com/86233755/164288753-9c91c588-4773-4da6-a00b-7b4475bae7ec.png)
and also the heart gets highlighted on hover but it doesnt register clicks - the number doesnt seem to increment currently in the dev version
